### PR TITLE
[stas-molchanov]fetch API practice

### PR DESCRIPTION
**Pull request by moskdev**

### CODE REVIEW

File: css-practice/task-3/js/apiParse.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/* global document*/
+function renderItem() {
```
Comment (dverbovyi) [inline css-practice/task-3/js/apiParse.js]: this function isn't used

File: css-practice/task-3/js/script.js [hunk 2]
```diff
@@ -1,113 +1,67 @@
...
+function fetchData() {
+  fetch('http://localhost:3000/api/blogs').then((res) => res.json()).then((res) => render(res.blogs, res.latest));
+}
 
-function toggleShow(e) {
-  const textClassList = e.currentTarget.nextElementSibling.classList;
+function render(blogs, latest) {
+  const blogsArr = [];
+  const latestArr = [];
+  const monthArr = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: It is better to create a separate function that copes with date.

File: css-practice/task-3/js/script.js [hunk 3]
```diff
@@ -1,113 +1,67 @@
...
-    active.previousElementSibling.classList.remove('prev-carou-item');
-    active.previousElementSibling.classList.add('next-carou-item');
-    active.previousElementSibling.previousElementSibling.classList.remove('prev-carou-item');
-    active.previousElementSibling.previousElementSibling.classList.add('active-carou-item');
-    return;
+  function renderUsual() {
+    blogsArr.forEach((el) => {
+      const blogCont = document.querySelector('.blog-container');
+      const dateObj = new Date(el.published);
+      const blogItem = document.createElement('div');
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: Empty line after 'const's is required for better readability.

File: css-practice/task-3/js/script.js [hunk 4]
```diff
@@ -1,113 +1,67 @@
...
+      blogItem.classList.add('blog-item');
+      blogItem.innerHTML = `
+      <div class="blog-img-container">
+        <img src="${el.previewImg}" alt="blog-image">
+        <div class="date"><span class="day">${dateObj.getDate()}</span> <span class="month cursive">${monthArr[dateObj.getMonth()]}</span></div>
+      </div>
+      <h3>${el.title}</h3>
+      <p>Consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: Blog description should be also rendered.

File: css-practice/task-3/js/script.js [hunk 5]
```diff
@@ -1,113 +1,67 @@
...
+      <hr>
+      <div class="activities cursive">
+        <img src="./img/eye.png" alt="views">${el.watched}
+        <img src="./img/comment.png" alt="comments">98
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: 'Comments' shouldn't be typed by hands, they should be rendered as well.

File: css-practice/task-3/js/script.js [hunk 6]
```diff
@@ -1,113 +1,67 @@
...
-    active.nextElementSibling.classList.remove('next-carou-item');
-    active.nextElementSibling.classList.add('prev-carou-item');
-    active.nextElementSibling.nextElementSibling.classList.remove('next-carou-item');
-    active.nextElementSibling.nextElementSibling.classList.add('active-carou-item');
-    return;
+  function renderLatest() {
+    latestArr.forEach((el) => {
+      const dateObj = new Date(el.published);
+      const blogCont = document.querySelector('.footer-blog-container');
+      const blogItem = document.createElement('div');
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: Empty line after 'const's.

File: css-practice/task-3/js/script.js [hunk 7]
```diff
@@ -1,113 +1,67 @@
...
+      blogItem.classList.add('footer-blog-item');
+      blogItem.innerHTML = `
+    <img src="./img/footer-blog-1.png" alt="blog-image">
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: Image source is stored in .json file.

File: css-practice/task-3/js/script.js [hunk 8]
```diff
@@ -1,113 +1,67 @@
...
+    <h3>${el.title}</h3>
+    <span class="cursive">
+      ${monthArr[dateObj.getMonth()]} ${dateObj.getDate()}, 2016
+    </span>
+      `;
+      blogCont.appendChild(blogItem);
+    });
   }
```
Comment (SofiMat) [inline css-practice/task-3/js/script.js]: Don't commit files that are not used (apiParse.js).

**Pull request merged**